include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -158,7 +161,9 @@ class OpKernelContext {
   const OrtValue* GetInputMLValue(int index) const;
   const OrtValue* GetImplicitInputMLValue(int index) const;
   OrtValue* GetOutputMLValue(int index);
-  OrtValue* OutputMLValue(int index, const TensorShape& shape);  // Creates the OrtValue* based on the shape, if it does not exist
+
+  // Creates the OrtValue* based on the shape, if it does not exist
+  OrtValue* OutputMLValue(int index, const TensorShape& shape, size_t nnz = 0);
```

Comment:
>nnz [](start = 70, length = 3)

'nnz' is a bit cryptic. can we use a more descriptive name in at least the headers, or document what it means in the comments?

Reply:
Added documentation. 'nnz' is a common acronym in the world of sparse-tensors / sparse-matrices and so it seems useful for us to adopt that terminology (though I have not been completely consistent about this).